src/lib/services/base-contract.service.ts

```diff
@@ -15,14 +15,16 @@ import {
   UniversalProfileInit__factory,
 } from '../..';
 import contractVersions from '../../versions.json';
-import { GAS_PRICE, NULL_ADDRESS } from '../helpers/config.helper';
+import { GAS_PRICE } from '../helpers/config.helper';
 import {
   deployBaseContract,
   getDeployedByteCode,
   waitForReceipt,
 } from '../helpers/deployment.helper';
 import { ContractNames as DigitalAssetContractNames } from '../interfaces/digital-asset-deployment';
 
+const { AddressZero } = ethersConstants;
+
 export function universalProfileBaseContractsDeployment$(
```

Comment:
FMI, What is the dollar at the end of the name of the variable ? :)

Reply:
I am not sure neither. @CallumGrindle ?

Reply:
Those are usually observables. It's a rxjs "standard" $ is a normal character inside of javascript and can be part of any identifiers.